apps/frontend/src/modules/surveys/components/studio/create/editor-panel/questions/attributes/EditFields/OptionsList.tsx

```diff
@@ -51,6 +51,7 @@ export const OptionsList = () => {
                                     defaultValue={option.label}
                                     onChange={(e) => {
                                         const newOptions = [...question.options];
+                                        console.log(newOptions);
```

Comment:
@afolasope Can you remove this log? Ensure you remove everything you added due to debugging before you push